docs/src/core-concepts.mdx

```diff
@@ -75,6 +75,9 @@ will fade away and you will mostly focus on the code within the braces.
 
 The above code is part of the [**Counter Example**](/examples/counter).
 
+For those who have experience with the JavaScript version of MobX, it's important to note that deep observability doesn't apply in the Dart version of MobX. This is because Dart is a statically typed language and there's no support for reflection in Flutter applications. Therefore, if a complex object is marked as observable then
+MobX will not be able to automatically track changes to the object's properties. Should change tracking need to enabled on the object's properties then the properties need to be marked as observable as well.
```

Comment:
> Should change tracking need to enabled on the object's properties then the properties need to be marked as observable as well.

I think this sentence should be worded differently.

Reply:
What's the reason behind your thought and what suggestions do you have? I could change it to "Should change tracking need to enabled on the object's properties then they should be marked as observable as well."

Reply:
I was thinking: "Should you need change tracking at the field level, its better to mark them with `@observable`"

Reply:
done, with a minor tweak to that sentence